_posts/2019-01-21-workshop.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Spinal Cord MRI Workshop at UCL"
```

Comment:
actually, do we want to make it a "6th" workshop? why not, since the name of the workshop does not include "annual" (thus, there could be more than one per year).

Reply:
It is up to you. But I would like to keep UCL in the name since UCL actively supports it.

Reply:
ok that's fine, let's leave it as it is then